oxidation/libparsec/crates/protocol/tests/test_handshake.rs

```diff
@@ -874,5 +874,26 @@ fn test_process_result_req_bad_outcome(
     assert_eq!(err, expected)
 }
 
+// Generated from Python implementation (Parsec v2.15.0+dev)
```

Comment:
Did you use the old Python or with the binding ?

Reply:
I used to `v2.15.0` tag. So I generated the hex code against the `NamedTuple` implementation in python.